app/src/main/java/com/sdp13epfl2021/projmag/UserTypeChoice.kt

```diff
@@ -0,0 +1,32 @@
+package com.sdp13epfl2021.projmag
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.widget.RadioButton
+import androidx.appcompat.app.AppCompatActivity
+import com.sdp13epfl2021.projmag.activities.ProfilePageActivity
+
+class UserTypeChoice : AppCompatActivity() {
+    lateinit var ProfButton: RadioButton
+    lateinit var StuButton: RadioButton
+
+    companion object{
+        var data: Boolean = false
+    }
```

Comment:
This meaning of this variable is not clear at all.

Reply:
I renamed the variable